packages/datasource/src/nav/activity/activity.data-zh.js

```diff
@@ -0,0 +1,4 @@
+export const backgroundImage = 'https://tidb.io/images/activity/banner.svg';
+export const buttonImage = 'https://tidb.io/images/activity/button.svg';
+export const text = '新功能上线！点击完成团队认证，解锁更多权益~';
+export const link = 'https://tidb.io/account/organization/new';
```

Comment:
We may think about how to remove redundant variables between translation files in the future.